src/cookies.js

```diff
@@ -0,0 +1,24 @@
+const cookiesDiv = document.querySelector('#cookies'); 
+
+// check if cookies are already accepted and hide the banner if true
+const cookiesAccepted = document.cookie.includes('cookiesAccepted=true');
+if (cookiesAccepted) {
+    cookiesDiv.style.display = 'none';
+} else {
+    cookiesDiv.style.display = 'flex';
+}
+// add event listener for DOMContentLoaded 
+document.addEventListener('DOMContentLoaded', () => {
+  const acceptBtn = document.querySelector('#accept-btn'); 
+
+  acceptBtn.addEventListener('click', () => {
+    // calculate the expiration date for the cookies (30 days from now)
+    const expires = new Date(Date.now() + 30 * 24 * 60 * 60 * 1000).toUTCString();
```

Comment:
Arată puțin ciudat asta. Cred că eu aș încerca fie să folosesc o librărie pentru calcule (de exemplu Day.js), ori ceva de genul:

```js
const expireDate = new Date();
expireDate.setDate(now.getDate() + 30);
const expires = expireDate.toUTCString();
```

Nu e greșit nici cum ai făcut tu.

Reply:
Aici asa am gasit pe Google, legat de cookies (multiplicare de miliseconds), dar metoda ta e muuult mai simpla si usor de inteles.
Multumesc frumos.